filters/filters.go

```diff
@@ -96,6 +96,12 @@ type FilterContext interface {
 
 	// Allow filters to create their own spans
 	ParentSpan() opentracing.Span
+
+	// Returns a clone of the FilterContext including a brand new request object.
```

Comment:
Can we please mention here that the stream of the request body is shared?

Reply:
I added more context here.